### ISSUE TITLE
can't reproduce the results reported in paper (CoNSep)

### ISSUE THREAD

**Issue by YangjiaqiDig** [vqdang/hover_net#159]
Hey, I need help here. I tried to reproduce your results on consep dataset. Please correct me if the setting is not same as yours:

1. extract patches: 540 * 540_164 * 164
2. first stage 50 epochs with msge loss weight = 2 and all the rests are 1. Lr starts from 1e-4 and decays by 0.1 every 25 epochs. The "pretrained" model is ImageNet-ResNet50-Preact_pytorch.tar. batch_seze 8
3. second stage 50 epochs with msge loss weight = 2 and all the rests are 1. Lr starts from 1e-4 and decays by 0.1 every 25 epochs. The "pretrained" model is -1 (previous phase), batch_size 4
4. I am running the 'original mode'

My results especially for F1_m(around 0.27) which is the first type are very bad. I also tried different settings but none of them could achieve the number reported in your paper.

**Comment by simongraham** [comment]
Hi @YangjiaqiDig please can you provide the F score that you achieve for all classes, not just `F1_m`

**Comment by YangjiaqiDig** [comment]
@simongraham 
<img width="835" alt="Screen Shot 2021-08-24 at 7 43 51 PM" src="https://user-images.githubusercontent.com/43737445/130703992-e7f7db9b-67c0-40e5-ae64-d449dfaeaa29.png">

**Comment by YangjiaqiDig** [comment]
<img width="835" alt="Screen Shot 2021-08-24 at 7 57 18 PM" src="https://user-images.githubusercontent.com/43737445/130704886-b3329279-f91f-4961-89c2-03739bcd45b4.png">
<img width="835" alt="Screen Shot 2021-08-24 at 7 57 09 PM" src="https://user-images.githubusercontent.com/43737445/130704887-42ac7144-c729-4c9a-a4f8-b4050ea8c981.png">
<img width="835" alt="Screen Shot 2021-08-24 at 7 57 00 PM" src="https://user-images.githubusercontent.com/43737445/130704888-85922abe-0125-4352-a198-69d008713ca5.png">
<img width="835" alt="Screen Shot 2021-08-24 at 7 56 47 PM" src="https://user-images.githubusercontent.com/43737445/130704889-bed73d4e-50d1-4182-952b-b18858c6767f.png">

Here are the tensorboard for stage 2

**Comment by simongraham** [comment]
Good to hear that your results are better with the correct patch extraction strategy. 

- We typically chose the best binary dice score as the selected epoch. However, this isn't necessarily the best thing to do. You may want to look at all dice scores to select your checkpoint.
- Yes, you should compare the the PyTorch results if using this repository.

**Comment by YangjiaqiDig** [comment]
Thank you!! I'm closing this now
